src/constants/colors.js

```diff
@@ -0,0 +1,13 @@
+const colors = {
+    green: '#17A1A2',
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
  green: '#17A1A2',
```